packages/SketchMorph2-Tests.package/M2DrawingComponentTests.class/instance/drawAmountOfDiagonalCircles..st

```diff
@@ -6,4 +6,4 @@ drawAmountOfDiagonalCircles: aNumber
 			userActionMoveDrawing: ((4 * x) + 2 ) asPoint;
 			userActionStopDrawing
 	]
```

Comment:
I'm not sure, but I don't think that is the squeak-like way of closing blocks.

Reply:
I can look into it, but I'm not sure I'll be able to automatically fix this. But I'll try.